src/ert/_c_wrappers/job_queue/qstat_proxy.sh

```diff
@@ -89,22 +89,26 @@ if [ $proxyage_seconds -gt $CACHE_INVALID ]; then
     exit 1
 fi
 
+# Read the contents into memory to avoid race conditions. Other
+# instances of this proxy might edit the proxyfile at any time.
+proxyfilecontents=$(cat $proxyfile)
+
 # Replicate qstat's error behaviour:
 if [ -n "$1" ]; then
-    grep -e "Job Id: ${1}" $proxyfile >/dev/null 2>&1 || {
-        echo "qstat: Unknown Job Id $1" && cat $proxyfile >&2 && exit 1;
+    echo "$proxyfilecontents" | grep -e "Job Id: ${1}" >/dev/null 2>&1 || {
```

Comment:
Why "" is needed?

Reply:
https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Word-Splitting